e2e_samples/fabric_dataops_sample/.envtemplate

```diff
@@ -16,5 +16,5 @@ export FABRIC_WORKSPACE_ADMIN_SG_NAME="The name of the Entra security groups wit
 # Fabric Capacity variables
 export EXISTING_FABRIC_CAPACITY_NAME="" # The name of an existing Fabric capacity. If this is empty, then a new capacity will be created.
 export FABRIC_CAPACITY_ADMINS="yourusername@yourdomain,sp_mi_object_id" # Comma separated list. When creating a new Fabric capacity, these users/apps would be added as capacity admin. For users, mention "userPrincipalName". For principals (sp/mi), mention "Object ID". Don't add spaces after the comma.
```

Comment:
For consistency, let's move all other comments to the previous line instead of at the end of line. For example:

# Fabric Capacity variables
## The name of an existing Fabric capacity. If this is empty, then a new capacity will be created.
export EXISTING_FABRIC_CAPACITY_NAME=""